### PR TITLE
Updated "Showcase" example

### PR DESCRIPTION
**Pull request by intonarumori**
Changes include:
- We are using `UITextField` instead of `UIView` for the base objects in Interface Builder.
This represents the proper usage of `SkyFloatingLabelTextField` in IB since it makes all the inherited IBDesignable properties from `UITextField` available in the property inspector.
- As a bonus you can also edit the text property by double clicking on the UI object (consistent with `UITextField`)
- Updated appearance in IB ignoring borderstyle since it's not supported by `SkyFloatingLabelTextField`
- Updated email validation trigger based on `UITextField` `editingChanged` action
- Updated email regex validation based on: http://emailregex.com

screen shot 2017-02-06 at 18 48 31

### CODE REVIEW

File: SkyFloatingLabelTextField/SkyFloatingLabelTextFieldExample/Example0/ShowcaseExampleViewController.swift [hunk 1]
```diff
@@ -213,10 +210,12 @@ class ShowcaseExampleViewController: UIViewController, UITextFieldDelegate {
         }
     }
     
-    func isValidEmail(str:String?) -> Bool {
-        let emailRegEx = "^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*$"
+    // MARK: - validation
+
+    // NOTE: source: http://emailregex.com
+    func validateEmail(_ candidate: String) -> Bool {
```
Comment (k0nserv) [inline SkyFloatingLabelTextField/SkyFloatingLabelTextFieldExample/Example0/ShowcaseExampleViewController.swift]: I'd actually prefer if we changed the email validation to only look for some characters and an @ sign.

Something like `.+@.+`. Correctly validating emails using regexes is notoriously difficult and I'd prefer if we didn't contribute to the idea that validating emails addresses with regex is a good idea.

Comment (intonarumori) [inline SkyFloatingLabelTextField/SkyFloatingLabelTextFieldExample/Example0/ShowcaseExampleViewController.swift]: I agree, and the previous code was even worse, but as I just included the source, and this is not mission critical code I think this is the easiest way for us.

Comment (intonarumori) [inline SkyFloatingLabelTextField/SkyFloatingLabelTextFieldExample/Example0/ShowcaseExampleViewController.swift]: Added the extended note explaining the regex validation and a better description of the source and further reading link.

Comment (k0nserv) [inline SkyFloatingLabelTextField/SkyFloatingLabelTextFieldExample/Example0/ShowcaseExampleViewController.swift]: The note is a good middleground and yes my regex would be valid for some invalid email address, but I think that's preferable to locking out people with valid addresses. The only real way to validate an email address is to send it an email after all.

Comment (k0nserv) [inline SkyFloatingLabelTextField/SkyFloatingLabelTextFieldExample/Example0/ShowcaseExampleViewController.swift]: The note is a good middleground and yes my regex would be valid for some invalid email address, but I think that's preferable to locking out people with valid addresses. The only real way to validate an email address is to send it an email after all.

**Comment by k0nserv** [comment]
The `NOTE:` was a good idea and this is still a big improvement over what we had before. 👍

**Pull request merged**